README.md

```diff
@@ -38,3 +38,45 @@ SiteSetting.sso_overrides_name = true
 SiteSetting.sso_overrides_email= true
 ```
 
+## Import from v5
+
+Loading the v5 database
+sudo -u postgres createdb c2corg
+sudo -u postgres psql -d c2corg < c2corg_v5_dump.sql
+
+Add missing columns:
+ALTER TABLE punbb_topics ADD COLUMN first_post_id integer default 0;
+UPDATE punbb_topics SET first_post_id = (select MIN(p.id) from punbb_posts as p where punbb_topics.id = p.topic_id);
+
+A discourse installation in developer mode is required:
+https://github.com/discourse/discourse/blob/master/docs/DEVELOPER-ADVANCED.md
+
+git clone https://github.com/discourse/discourse
+cd discourse
+(gem install bundle --user-install)
+bundle install --path ~/gempath
+
+A database user with enough rights is necessary:
+sudo -u postgres createuser -s -P  discourse_import
+
+Tables need to be created:
+PGHOSTADDR=127.0.0.1 PGUSER=discourse_import  PGPASSWORD=discourse_import bundle exec rake db:create db:migrate
+
+
+
+Migration:
+date; DATABASE_URL=postgres://discourse_import:discourse_import@127.0.0.1/discourse_development bundle exec ruby  script/import_scripts/punbb.rb; date
+
+## Testing (with standalone, non sso instance)
+
+Set 'c2cc2cc2' password to everyone:
```

Comment:
Is it actually ``c2cc2cc2`` (OK for me, just want to make sure that's not a typo)?